### PR TITLE
Azure Maps Elevation SDK Deprecation

### PR DESCRIPTION
**Pull request by khmic5**
Update CSV file for deprecation

### CODE REVIEW

File: _data/releases/latest/java-packages.csv [hunk 1]
```diff
@@ -7,7 +7,7 @@
 "azure-data-appconfiguration","com.azure","1.4.3","","App Configuration","App Configuration","appconfiguration","","","client","true","","03/18/2023","01/07/2020","active","","","","","","",""
 "azure-security-attestation","com.azure","1.1.11","","Attestation","Attestation","attestation","","","client","true","","03/18/2023","02/08/2022","","","","","","","",""
 "azure-messaging-eventhubs-checkpointstore-blob","com.azure","1.16.4","1.17.0-beta.1","Azure Blob Storage Checkpoint Store","Event Hubs","eventhubs","","","client","true","","03/18/2023","02/04/2020","active","","","","","","",""
-"azure-maps-elevation","com.azure","","1.0.0-beta.1","Azure Maps Elevation","Maps","maps","","","client","true","","","","","","","","","","",""
+"azure-maps-elevation","com.azure","","1.0.0-beta.1","Azure Maps Elevation","Maps","maps","","","client","true","","","","deprecated","05/05/2023","","Get Elevations","aka.ms/azsdk/java/migrate/azure-maps-elevation","","",""
```
Comment (weshaggard) [inline _data/releases/latest/java-packages.csv:10]:
```suggestion
"azure-maps-elevation","com.azure","","1.0.0-beta.1","Azure Maps Elevation","Maps","maps","","","client","true","","","","deprecated","05/05/2023","","Get Elevations","htps://aka.ms/azsdk/java/migrate/azure-maps-elevation","","",""
```

Is the plan to not have an SDK at all?

Comment (dubiety) [reply _data/releases/latest/java-packages.csv:10]: Yes. We deprecate the service and also don't want to provide the SDK for elevation service.

**Review by dubiety — APPROVED** [review]

**Review by mario-guerra — CHANGES_REQUESTED** [review]
it looks like you've specified a deprecation date that has already passed. As per the [Azure Retirement Policy,](https://eng.ms/docs/cloud-ai-platform/azure-core/azure-core-pm-and-design/trusted-platform-pm-karimb/service-lifecycle-and-actions-team/service-lifecycle-actions-team/APEX/media/overview_retirements) the date needs to be at least 12 months out for Track 1 packages and 36 months out for Track 2 packages.

From the guidance:
_For Track 1 SDK libraries, product teams may announce "end of support in 1 year" after a replacement Track 2 library is available. Support during the 1 year may be limited to just critical security fixes. Track 2 libraries must follow the standard 3-year notification._

**Comment by khmic5** [comment]
Hi Mario, thank you for the comment. According to the Azure SDK deprecation policy, https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/192/Azure-SDK-deprecation-strategy?anchor=...less-than-one-year-old, it looks like beta less than a year old can be marked deprecated at any time. If this looks ok to you we will continue with the pr!

**Pull request merged**